expr/helper/helper.go

```diff
@@ -183,18 +183,23 @@ func SummarizeValues(f string, values []float64) (float64, bool) {
 		if len(values) > 0 {
 			rv = values[len(values)-1]
 		}
+	case "count":
+		rv = float64(len(values))
 
 	default:
-		f = strings.Split(f, "p")[1]
-		percent, err := strconv.ParseFloat(f, 64)
-		if err == nil {
-			return Percentile(values, percent, true)
+		if strings.HasPrefix(f, "p") {
```

Comment:
This check seems partial. "p" being a prefix does not mean we have a correct percentile. In case we get something like "puff", the returning error will be misleading. I suggest we change the check to something like `^p\d\d?$`.

What do you think?

Reply:
I like this idea!
I used `^p\d` because it seems we can have float percentiles here. For example: `p99.99'